src/llmcompressor/transformers/finetune/training_args.py

```diff
@@ -32,6 +32,12 @@ class TrainingArguments(HFTrainingArgs):
             )
         },
     )
+    calibration_batch_size: int = field(
```

Comment:
Oneshot will not depend on training_args in the follow up PR, so this will be moved once that lands. 
For now its ok since oneshot uses training_args

Reply:
Which argument set should this exist on @horheynm?